node/bft/storage-service/src/memory.rs

```diff
@@ -124,6 +130,23 @@ impl<N: Network> StorageService<N> for BFTMemoryService<N> {
                 }
             }
         }
+        // Inserts the aborted transmission IDs.
+        for aborted_transmission_id in aborted_transmission_ids {
+            // Retrieve the transmission entry.
+            match aborted_transmission_ids_lock.entry(aborted_transmission_id) {
```

Comment:
If Pratyush would see this he would tell you to use [or_insert](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html#method.or_insert)